test/integration/service_manual_guide_test.rb

```diff
@@ -1,6 +1,28 @@
 require 'test_helper'
 
 class ServiceManualGuideTest < ActionDispatch::IntegrationTest
+  test "shows the time it was saved and hasn't been published yet" do
```

Comment:
Yeah, it's the use of the word 'and' makes it sound like you're _testing_ that it hasn't been published yet. `shows the time it was saved if it hasn't been published yet` would also work.